javagen/src/main/java/com/azure/autorest/model/clientmodel/ClientMethod.java

```diff
@@ -273,8 +273,8 @@ public void addImportsTo(Set<String> imports, boolean includeImplementationImpor
         if (settings.isLowLevelClient()) {
             imports.add("com.azure.core.http.HttpMethod");
             imports.add("com.azure.core.http.rest.Response");
-            imports.add("com.azure.core.http.RequestOptions");
-            imports.add("com.azure.core.util.BinaryData");
+            ClassType.RequestOptions.addImportsTo(imports, true);
+            ClassType.BinaryData.addImportsTo(imports, true);
```

Comment:
What's the difference? `implementationImports` in builder is never used in project.